engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EmailAggregationRepository.java

```diff
@@ -31,9 +31,10 @@ public boolean addEmailAggregation(EmailAggregation aggregation) {
     }
 
     public List<EmailAggregation> getEmailAggregation(EmailAggregationKey key, LocalDateTime start, LocalDateTime end) {
-        String query = "FROM EmailAggregation WHERE accountId = :accountId AND bundleName = :bundleName AND applicationName = :applicationName AND created > :start AND created <= :end ORDER BY created";
+        String query = "FROM EmailAggregation WHERE accountId = :accountId AND orgId = :orgId AND bundleName = :bundleName AND applicationName = :applicationName AND created > :start AND created <= :end ORDER BY created";
         return statelessSessionFactory.getCurrentSession().createQuery(query, EmailAggregation.class)
                 .setParameter("accountId", key.getAccountId())
+                .setParameter("orgId", key.getOrgId())
```

Comment:
We should only use it if it's not null. 
We might want to initially only have it as part of the APIs, but not really use it or only use if it is set.

Reply:
Changed now. I will go through all the changes once again and see if there is another piece using the orgId.